microsoft/botframework-solutions
# [TypeScript][BotBuilder-Libs] Remove unused dependencies

## Purpose

Removed unused dependencies from `package.json` in the following project:
  - `botbuilder-solutions`
  - `botbuilder-skills`

## Tests

1. Go to  `.\lib\typescript` and open a PowerShell terminal
1. Run `rush install` to install dependencies
1. Run `rush build` to build both projects
1. Run `rush test` to run the tests
1. Verify that the test run successfully
